### PR TITLE
Python3 compatibility

### PR DESCRIPTION
**Pull request by berdario**

### CODE REVIEW

File: RangeHTTPServer.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/python
 '''
-Use this in the same way as Python's SimpleHTTPServer:
+Use this in the same way as Python's http.server:
```
Comment (danvk) [inline RangeHTTPServer.py]: How about `http.server / SimpleHTTPServer`? It works with either Python 2 or 3, after all.

File: RangeHTTPServer.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
   python -m RangeHTTPServer [port]
 
-The only difference from SimpleHTTPServer is that RangeHTTPServer supports
+The only difference from HTTPServer is that RangeHTTPServer supports
```
Comment (danvk) [inline RangeHTTPServer.py]: `http.server / SimpleHTTPServer`

**Comment by danvk** [comment]
Thanks for the change.

FWIW, I stopped using this once I discovered NPM's [http-server](https://www.npmjs.com/package/http-server), which does everything this does and much more.

**Comment by berdario** [comment]
I see

I'm using it for the test suite of this [very small project](https://github.com/berdario/resumable-urlretrieve/blob/master/test/test_resumable.py) (I actually haven't written any single line yet... I just added the tests)

Being able to whip up a daemon thread (so it'll be tore down automatically) in the same process in which you're running the integration tests is quite nice (instead of having to rely on another tool with a completely separate runtime... which would be a pain to install, deploy, document, etc...)

Are you going to merge this PR?

**Comment by danvk** [comment]
Sure, just make the two changes I requested and I'll merge it in.

**Comment by berdario** [comment]
done

**Comment by danvk** [comment]
Thanks! Merged.

**Comment by berdario** [comment]
Thanks for your rapidity

**Comment by berdario** [comment]
Adding a new release on pypi would be nice (in the meanwhile I'm just pointing pip to github)

**Comment by danvk** [comment]
Hi there, this seems to have broken Python 2 compatibility:

```
./RangeHTTPServer.py 8080
Traceback (most recent call last):
  File "./RangeHTTPServer.py", line 127, in <module>
    http_server.test(HandlerClass=RangeRequestHandler, port=args.port, bind=args.bind)
TypeError: test() got an unexpected keyword argument 'port'
```

Mind fixing this? I'll revert in the meantime.

**Comment by berdario** [comment]
Weird, I never received email notification for this

I see that #12 has been opened in the meanwhile

I'll try to have a look at it soon

**Pull request merged**